### ISSUE TITLE
After submitting an app, the success view should give you your magic link

### ISSUE THREAD

**Issue by hstove** [blockstack/app.co#521]

**Comment by hstove** [comment]
Just pushed this. When a user submits a new app, and:

- the 'authentication' field is `Blockstack`
- the 'category' field is NOT `Sample Blockstack Apps`

They will see their magic link:

![screenshot](https://cl.ly/f28752e5463c/Image%202019-06-17%20at%207.05.18%20AM.png)

I have this only showing for Blockstack apps, since this flow is currently only for app mining. We can change up the logic here if/when we make this more of a general-purpose maker portal.

**Comment by markmhx** [comment]
The submit button shows "Loading" briefly upon submit then goes back to "Submit" when testing:

<img width="1138" alt="Screen Shot 2019-06-17 at 17 24 11" src="https://user-images.githubusercontent.com/28991/59616468-c61ab500-9124-11e9-9485-714f7ab48ffc.png">

**Comment by markmhx** [comment]
Note that I did receive the "Your App Submission Has Been Received" email despite the above error.

**Comment by hstove** [comment]
I believe there is an unrelated error caused by using an app name that already exists. Did you use an app name like "testing"? If so, try an app name that is more unique. If you can confirm the issue is a duplicate name, please file a separate ticket.

**Comment by timstackblock** [comment]
cannot test this until I get admin privileges

**Comment by markmhx** [comment]
Tried again with a more unique app name but same result

<img width="433" alt="Screen Shot 2019-06-18 at 16 54 39" src="https://user-images.githubusercontent.com/28991/59695000-c7fa7c00-91e9-11e9-8a34-dd6c77f2d895.png">

**Comment by timstackblock** [comment]
Blocked by this infinite loop during sign in @markmhx are you accessing this I cant get pass sign in I keep getting the issue in this video https://github.com/blockstack/app.co/issues/536

**Comment by markmhx** [comment]
Yea I can access this but have encountered the same problem as you on my laptop (can't remember the browser). Have you tried a different browser for testing?

**Comment by timstackblock** [comment]
I get this in Chrome, I get different errors in Safari and FF I logged those as well

**Comment by hstove** [comment]
Hey @markmhx , I've tried a few different things and can't reproduce this issue. I try submitting all kinds of apps, with Blockstack auth and otherwise, with no success. Maybe can you tell me exactly what details you enter that cause this?

I also fixed the issue of apps with a duplicate name, but it sounds like that's not the issue anyways.

Also, per your screenshot, you have to have 'Blockstack' as auth to see this new screen.

**Comment by markmhx** [comment]
I just tried again and couldn't reproduce the bug I encountered earlier (in Chrome on MacOS). 

The success screen shows up properly for me now:

<img width="741" alt="Screen Shot 2019-06-21 at 17 32 14" src="https://user-images.githubusercontent.com/28991/59933876-8ddbe500-944a-11e9-9582-472be9fe8a54.png">

**Comment by timstackblock** [comment]
I accessed this by pating in the URL and going directly to https://app-co-staging-pr-519.herokuapp.com/submit in my browser this works but how can I access this with out going directly to the link.

What is the normal flow to get to this page I would like to test that before I close it out.

@hstove @markmhx

**Comment by timstackblock** [comment]
Ignore the comment above this works as expected, closing this out now.
